src/app/(with-main-header)/MobileHeader.tsx

```diff
@@ -54,16 +101,71 @@ export default function MobileHeader() {
             blurDataURL={"../assets/images/Logo.svg"}
           />
         </Link>
-        <div className="flex gap-2.5">
-          <Link href={"/chatting"} onClick={closeSidebar}>
-            <Icon MuiIcon={SendRoundedIcon} />
-          </Link>
-          <button>
-            <Icon MuiIcon={NotificationsRoundedIcon} />
-          </button>
-        </div>
+        {isLoggedIn ? (
+          <div className="flex gap-2.5">
```

Comment:
이 부분은 `isLoggedIn`이 false일 때 결국 null을 반환하기 때문에, `isLoggedIn`이 true일 때만 보여주는 게 더 보기 편할 것 같아요!

```suggestion
        {isLoggedIn && (
          <div className="flex gap-2.5">
```